lib/gpu_memory_service/common/protocol.py

```
@@ -0,0 +1,653 @@
+            msg = decode_message(msg_data)
+            return msg, -1, remaining
+
+    # Need to receive more data
+    if raw_sock is not None:
+        # Use recvmsg for potential FD receiving
+        ancillary_size = socket_module.CMSG_SPACE(struct.calcsize("i"))
+
+        def do_recvmsg():
+            return raw_sock.recvmsg(65540, ancillary_size)
+
+        raw_msg, ancdata, flags, _ = await loop.run_in_executor(None, do_recvmsg)
```

Ruff (0.14.10)

Unpacked variable `flags` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)